app/services/streamlabels/definitions.ts

```diff
@@ -16,10 +16,11 @@ export interface IStreamlabelDefinition {
 }
 
 export interface IStreamlabelSettingsDefinition {
-  format: { tokens: string[]; };
+  format?: { tokens: string[]; };
   item_format?: { tokens: string[]; };
   item_separator?: { tokens: string[]; };
-  includeResubsOption?: boolean;
+  settingsStat?: string;
+  settingsWhitelist?: string[];
```

Comment:
Wow, the all fields are optional?

Reply:
Yeah.  It very much depends on the type of streamlabel is.